src/aims_ui/page_uprn.py

```diff
@@ -38,6 +38,9 @@ def uprn():
       all_user_input,
   )
 
+  if result == 'error_connecting':
```

Comment:
Likewise, this looks like more error-checking code that could just be a `catch`.